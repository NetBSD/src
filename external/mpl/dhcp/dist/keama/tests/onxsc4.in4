# on executable statement construct

# empty configs are not accepted by Kea
default-lease-time 1800;

# empty on statement
on expiry;

# another one
on commit or release {
    execute ("myscript", packet(5, 2));
}
