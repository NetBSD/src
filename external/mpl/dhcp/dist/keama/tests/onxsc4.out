{
  # on executable statement construct
  # empty configs are not accepted by Kea
  "Dhcp4": {
    "valid-lifetime": 1800
//  # empty on statement
//  "statement": {
//    "on": {
//      "condition": "expiry"
//    }
//  }
//  # another one
//  "statement": {
//    "on": {
//      "condition": "commit or release",
//      "body": [
//        {
//          "execute": {
//            "command": "myscript",
//            "arguments": [
//              {
//                "packet": {
//                  "offset": 5,
//                  "length": 2
//                }
//              }
//            ]
//          }
//        }
//      ]
//    }
//  }
  }
}
